### DISCUSSION TITLE
Could you help move my account to the correct team?

### DISCUSSION THREAD

**Discussion by mengcz13** [PaddlePaddle/PaddleSpatial#170] [BaiduKDDCup2022] [answered]
Dear organizers,

Could you help move the account associated with this GitHub (@mengcz13) account to the team "AIStudio2144990Team" and remove the user with an unidentifiable username from the team?

When I used the invitation link from the team leader, the system just asked me for some information without anything related to the account. When I logged out and logged in again, the system asked for a GitHub account, and I found myself in a newly created team with only me after logging in. Meanwhile, the desired team got one more member with an unidentifiable username (as shown in the following figure).

<img width="272" alt="image" src="https://user-images.githubusercontent.com/12379241/173509435-2b7a0ab2-5ded-4b18-b3e2-b4e082ae073e.png">

Thanks!

**Comment by ramber1836** [comment, marked as answer]
Hi there, you can ask the team leader of the correct team to remove the unidentifiable user, then let the team leader invite you again to rejoin this team (AIStudio2144990Team) with your correct user name as long as you didn't submit any code with the newly created team.
